relay-filter/src/interface.rs

```diff
@@ -31,6 +31,9 @@ pub trait Filterable {
 
     /// The user agent of the client that sent the data.
     fn user_agent(&self) -> Option<&str>;
+
+    /// The header for the given `header_name`. Used for localhost filtering.
```

Comment:
The usage can be an example, otherwise this is very likely to be outdated.

Some more info would be good what value is expected here, this is not the header from the request being sent to Relay, but from request metadata contained in the item.